CenterForOpenScience/osf.io
# [#PLAT-48]Embedding identifiers in preprint list endpoint is broken

## Purpose
Fix the problem that the embedding identifiers is broken


## Changes
change the view category to be the right category and make the reverse work correctly


## QA Notes
1. create a new preprint A
2. go to staging-api.osf.io/v2/preprints/<A._id>/?embed=identifiers
3. check there is no 502 on the page and check the "data": "embeds": "identifiers": "link":"self" link is correct like "api.osf.io/v2/preprints/<A._id>/identifiers/<id>" and not like  "api.osf.io/v2/identifiers/<A._id>/identifiers/<id>"

CORRECT version:

<img width="1076" alt="screen shot 2018-01-22 at 3 40 09 pm" src="https://user-images.githubusercontent.com/4974056/35243360-8faaadb8-ff8a-11e7-8530-2e791ef78f12.png">

WRONG version:
<img width="1100" alt="screen shot 2018-01-22 at 3 40 46 pm" src="https://user-images.githubusercontent.com/4974056/35243386-a5007436-ff8a-11e7-8400-058e76f3fdd0.png">

4. create a project and then repeat step 2-3 with the node's api url


## Side Effects



## Ticket
https://openscience.atlassian.net/browse/PLAT-48
